packages/run-protocol/src/vaultFactory/vaultFactory.js

```diff
@@ -24,15 +24,23 @@ import {
   makeVaultDirectorParamManager,
   LIQUIDATION_INSTALL_KEY,
   LIQUIDATION_TERMS_KEY,
+  MIN_INITIAL_LOAN_KEY,
 } from './params.js';
 import { makeVaultDirector } from './vaultDirector.js';
 
 /**
- * @typedef {ZCF<GovernanceTerms<{}> & {
+ * @typedef {{
+ *   LiquidationTerms: ParamRecord<'unknown'>,
+ *   LiquidationInstall: ParamRecord<'installation'>,
+ *   MinInitialLoan: ParamRecord<'amount'>,
```

Comment:
Don't use "loan" for this.

Reply:
An alternative to renaming it is to structure it differently; e.g., "MinCreationFee" and then either always charge at least that, or reject any loan that doesn't have at least that much in fees.

Reply:
In the issue (#4967) I asked about how to structure the restriction. I think the limit that makes the most sense to users and is straightforward for us to implement is a limit on how much the user borrows. If we need to discuss alternatives further, the context is in that issue.